jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -167,6 +191,11 @@ def active_image_layer(self):
 
         return visible_layers[-1]
 
+    def _on_global_display_unit_changed(self, msg):
+        # TODO: change slice_values to cached_property and clear cache
+        # del self.slice_values
+        self.slice_values
```

Comment:
@kecnry I'm sure this is incorrect but I'm not sure how to clear the cache of `slice_values` when it returns an `np.ndarray` if I change `slice_values` to a `cached_property`.

Reply:
you just want to clear the cache when the display unit is changed?